### PR TITLE
 [Doctest] Add `configuration_resnet.py`

### PR DESCRIPTION
**Pull request by daspartho**
Add `configuration_resnet.py` to `utils/documentation_tests.txt` for doctest.

Based on issue #19487

@ydshieh could you take a look at it?
Thanks =)

### CODE REVIEW

File: src/transformers/models/resnet/configuration_resnet.py [hunk 1]
```diff
@@ -65,7 +65,7 @@ class ResNetConfig(PretrainedConfig):
 
     >>> # Initializing a ResNet resnet-50 style configuration
     >>> configuration = ResNetConfig()
-    >>> # Initializing a model from the resnet-50 style configuration
+    >>> # Initializing a model (with random weights) from the resnet-50 style configuration
```
Comment (ydshieh) [inline src/transformers/models/resnet/configuration_resnet.py:68]: Can we have some new empty lines, just the ones you can find in other config files 🙏 , thanks

Comment (daspartho) [reply src/transformers/models/resnet/configuration_resnet.py:69]: @ydshieh made the suggested changes; good to go now :)

**Review by ydshieh — APPROVED** [review]
Thank you for giving me a chance to not just say `thank you` in this PR :-)

File: src/transformers/models/resnet/configuration_resnet.py [hunk 2]
```diff
@@ -68,7 +68,7 @@ class ResNetConfig(PretrainedConfig):
 
     >>> # Initializing a model (with random weights) from the resnet-50 style configuration
     >>> model = ResNetModel(configuration)
-    
+
```
Comment (ydshieh) [inline src/transformers/models/resnet/configuration_resnet.py:71]: You are faster than my comment

Comment (daspartho) [reply src/transformers/models/resnet/configuration_resnet.py:71]: Noticed a few tests failing so ran it; should work nicely now 😄

**Pull request merged**